packages/link/lib/TextLink.jsx

```diff
@@ -12,13 +12,17 @@ import * as styles from './text-link.module.scss';
  */
 export default function TextLink({icon, iconBefore, text, ...linkProps}) {
   if (icon) {
-    icon = React.cloneElement(icon, {key: 'icon'});
+    icon = React.cloneElement(icon, {
+      // Icons should be hidden from screenreaders if text is available.
+      'aria-hidden': !!text,
```

Comment:
behavior change: hide the icon from screenreaders if text is present. thank you @epeach for reminding me about this (and i worked this change into the TextLink unit tests, which i thought you'd find interesting)!